pkg/querier/queryrange/results_cache.go

```diff
@@ -26,12 +26,14 @@ import (
 type ResultsCacheConfig struct {
 	CacheConfig       cache.Config  `yaml:"cache"`
 	MaxCacheFreshness time.Duration `yaml:"max_freshness"`
+	SplitInterval     time.Duration `yaml:"split_interval"`
```

Comment:
Would be better to keep consistency with the CLI flag, so `cache_split_interval` instead of `split_interval`.